README.md

```diff
@@ -15,4 +15,4 @@ In order to use a local build for your local projects, you'll need to publish it
 maven local repo. To do so, open up a terminal in this project's root dir and
 run: `./gradlew publishToMavenLocal`. After that, you can apply the agent into an Android
 application project (that uses [mavenLocal()](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api.artifacts.dsl/-repository-handler/maven-local.html) as one of its dependency repositories)
-by following the [Getting started](https://www.elastic.co/guide/en/apm/agent/android/current/setup.html)" guide and using the version defined [here](gradle.properties).
+by following the [Getting started](https://www.elastic.co/guide/en/apm/agent/android/current/setup.html) guide and using the version defined [here](gradle.properties).
```

Comment:
@LikeTheSalad Could we update this to the new version?

Reply:
This is a great point, it's updated now 👍